src/structs/lepton_encoder.rs

```diff
@@ -728,10 +748,10 @@ fn roundtrip_large_coef() {
 fn roundtrip_random_seed() {
     use rand::Rng;
 
-    // the 22 seed is a choice that doesn't overflow the DC coefficient
```

Comment:
dont change the seeds to make sure theres no change in behavior

Reply:
This seed fails providing the same hashes in scalar and 16-bit path. I'm investigating this.

Reply:
I've traced the difference to that bug: [here](https://github.com/microsoft/lepton_jpeg_rust/blob/main/src/structs/probability_tables.rs#L360) is quantization coefficient is taken as i16.